### ISSUE TITLE
Can not train the VGG-16 OR Resnet-50 Normally In Keras（Loss Stable）

### ISSUE THREAD

**Issue by ChawDoe** [keras-team/keras#11145]
The main code is as follows:
![issue](https://user-images.githubusercontent.com/25716030/45582351-b98cd600-b8e0-11e8-9234-3165cab6b717.png)
![issue2](https://user-images.githubusercontent.com/25716030/45582479-581a3680-b8e3-11e8-89f4-a786237354bd.png)
![issue3](https://user-images.githubusercontent.com/25716030/45582480-5badbd80-b8e3-11e8-9e5a-061db382a65b.png)

I make a model which has 3 input and make vgg-16 be its feature extractor.
I want to do similarity measure after feature extraction.
But I failed to train it on the official vgg model and successfully train it on my own sequential CNN model.
The loss is always const.
I change the lr to 1 but the problem always exists.
I think it's a bug in keras.
